### PR TITLE
Setup register sandbox environment

### PR DESCRIPTION
**Pull request by felixtheflex**
### Context

Create Sandbox environment

### Changes proposed in this pull request

Create Sandbox environment

### CODE REVIEW

**Comment by felixtheflex** [comment]
[Register sandbox environment](https://trello.com/c/3b3uCQGo/917-register-sandbox-environment)

File: terraform/workspace-variables/sandbox.tfvars [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# PaaS
+paas_app_environment             = "sandbox"
+env_config                       = "sandbox"
+paas_space_name                  = "sandbox"
```
Comment (vigneshmsft) [inline terraform/workspace-variables/sandbox.tfvars:4]: this should be `prod` ?

File: terraform/workspace-variables/sandbox.tfvars [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+statuscake_alerts = {
+
+  alert = {
+    website_name   = "register-sandbox"
+    website_url    = "https://sandbox.register-trainee-teachers.education.gov.uk/ping"
+    test_type      = "HTTP"
+    check_rate     = 60
+    contact_group  = [188603]
+    trigger_rate   = 0
+    custom_header  = "{\"Content-Type\": \"application/x-www-form-urlencoded\"}"
```
Comment (saliceti) [inline terraform/workspace-variables/sandbox.tfvars:24]: Could you please simplify and use the defaults? Look at https://github.com/DFE-Digital/publish-teacher-training/pull/1513

File: terraform/workspace-variables/sandbox.tfvars [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# PaaS
+paas_app_environment             = "sandbox"
+env_config                       = "sandbox"
+paas_space_name                  = "sandbox"
+paas_postgres_service_plan       = "small-ha-11"
```
Comment (saliceti) [inline terraform/workspace-variables/sandbox.tfvars:5]: postgres and redis don't need ha

File: config/settings/sandbox.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# The url for this app, also used by `dfe_sign_in`
+base_url: https://www.register-trainee-teachers.education.gov.uk
```
Comment (saliceti) [inline config/settings/sandbox.yml:2]: Isn't it `https://www.register-trainee-teachers.education.gov.uk` ?

File: terraform/workspace-variables/sandbox.tfvars (hunk 1 above)
Comment (felixtheflex) [reply terraform/workspace-variables/sandbox.tfvars]: fixed up...

File: terraform/workspace-variables/sandbox.tfvars (hunk 2 above)
Comment (felixtheflex) [reply terraform/workspace-variables/sandbox.tfvars]: fixed..

File: terraform/workspace-variables/sandbox.tfvars (hunk 3 above)
Comment (felixtheflex) [reply terraform/workspace-variables/sandbox.tfvars]: done...

File: config/settings/sandbox.yml (hunk 4 above)
Comment (felixtheflex) [reply config/settings/sandbox.yml]: done..

File: terraform/workspace-variables/sandbox.tfvars [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# PaaS
+paas_app_environment             = "sandbox"
+env_config                       = "sandbox"
+paas_space_name                  = "bat-prod"
+paas_postgres_service_plan       = "small-11"
+paas_redis_service_plan          = "tiny-4_x"
```
Comment (saliceti) [inline terraform/workspace-variables/sandbox.tfvars:6]: Can we use micro 5?

**Review by saliceti — APPROVED** [review]

File: terraform/workspace-variables/sandbox.tfvars (hunk 5 above)
Comment (felixtheflex) [reply terraform/workspace-variables/sandbox.tfvars]: done..

**Pull request merged**